src/client/components/input/select2.js

```diff
@@ -31,10 +32,21 @@ if (typeof window !== 'undefined') {
 }
 
 class Select extends React.Component {
+
+	/**
+	 * Invoked after component mount that initializes select2
```

Comment:
I'd suggest avoiding comments that describe what is calling a function; that's liable to change. Instead, describe what the function does.